std/experimental/allocator/common.d

```diff
@@ -82,10 +84,11 @@ struct Ternary
     }
 }
 
+@safe @nogc nothrow pure
 unittest
 {
     alias f = Ternary.no, t = Ternary.yes, u = Ternary.unknown;
-    auto truthTableAnd =
```

Comment:
I also prefer `auto`, but array literals are not statically allocated and the whole unittest will become non-`@nogc`.
There was a proposal for the syntax `[1, 2, 3]s` and `auto[$] arr = [1, 2, 3]`, however it was rejected :(